nodes/parachain/src/command.rs

```diff
@@ -149,14 +149,24 @@ pub(crate) fn run() -> sc_cli::Result<()> {
 		Some(Subcommand::ExportGenesisWasm(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
 			runner.sync_run(|_config| {
-				let (chain_spec_id, _) = get_selected_chainspec(&cli)?;
+				let (chain_spec_id, _) = get_selected_chainspec(&cmd.shared_params)?;
 				let spec = cli.load_spec(chain_spec_id.as_str())?;
 
 				cmd.run(&*spec)
 			})
 		}
 		Some(Subcommand::Benchmark(cmd)) => {
-			let (_, runtime) = get_selected_chainspec(&cli)?;
+
+			let shared_params = match cmd {
```

Comment:
So there's no trait implementation that can be used on the `cmd` object to get the `shared_params` property? What about the `SubstrateCli` trait?